app/controllers/v0/sessions_controller.rb

```diff
@@ -33,6 +33,9 @@ def saml_callback
       if @saml_response.is_valid? && persist_session_and_user
         async_create_evss_account(@current_user)
         redirect_to SAML_CONFIG['relay'] + '?token=' + @session.token
+
+        obscure_token = Session.obscure_token(@session.token)
+        Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
```

Comment:
I'm really not sure where we're supposed to draw the line with the PII here. Since the whole point of the thing is to be able to look up requests related to a user, it stands to reason that we ought to tolerate some level of leakage. If we scrub all of the connection to their account then it really won't do much good. @jcmeloni do you have an opinion on this related to Bill's last comment?